src/AutoRest.CSharp/Common/Output/Models/ConstructorSignature.cs

```diff
@@ -6,5 +6,5 @@
 
 namespace AutoRest.CSharp.Output.Models
 {
-    internal record ConstructorSignature(string Name, string? Description, MethodSignatureModifiers Modifiers, IReadOnlyList<Parameter> Parameters, ConstructorInitializer? Initializer = null) : MethodSignatureBase(Name, Description, Modifiers, Parameters);
+    internal record ConstructorSignature(string Name, string? Description, string? Summary, MethodSignatureModifiers Modifiers, IReadOnlyList<Parameter> Parameters, ConstructorInitializer? Initializer = null) : MethodSignatureBase(Name, Description, Summary, Modifiers, Parameters);
```

Comment:
`Summary` should be before `Description` in the sequence. See the official document and example: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d312-remarks

Reply:
This line is about variable sequence. Are you saying the output sequence?